### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Edvinas01**
Buckle up

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,7 @@ yarn-error.log*
 
 .env
 *.css
+
+# IntelliJ IDEA
```
Comment (milasevicius) [inline .gitignore]: Why your IDE cofig files are not in your local .globalgitignore file?

File: src/components/app/App.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import Places from '../place/Places'
+import {getMessage} from '../../util/localization'
+
+import 'bootswatch/dist/materia/bootstrap.min.css'
+import './app.css'
+
+class App extends Component {
+
+  constructor(props) {
+    super(props);
```
Comment (milasevicius) [inline src/components/app/App.js]: Why ; is not constistant?

File: src/components/app/App.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react'
```
Comment (milasevicius) [inline src/components/app/App.js]: Filenames should have this structure `ClassName/index.js` to have cleaner imports.

File: src/components/place/Places.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+      .catch(msg => console.error(msg))
+      .then(() => this.setProgress(-1));
+  }
+
+  /**
+   * Adjust distance information for each place and sort them.
+   */
+  adjustPlaces(places, location) {
+    return places
+      .map(p => Object.assign(p, {
```
Comment (milasevicius) [inline src/components/place/Places.js]: You could use a spread operator here:

`{{distance: distanceBetween(location, p.location)}, ...p}`

File: src/components/search/Search.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+import {
+  Input,
+  FormGroup,
+  Form
+} from 'reactstrap';
+
+import {getMessage} from "../../util/localization";
```
Comment (milasevicius) [inline src/components/search/Search.js]: Inconsistent double quotes

File: src/util/api.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import 'whatwg-fetch';
```
Comment (milasevicius) [inline src/util/api.js]: Rename directory `util` to plural

File: src/util/localization.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Change locale.
+ */
+export const setLocale = (name) =>
+  currentLocale = locales[name.toLowerCase().trim()];
+
+/**
+ * Get localized message using locale message key.
+ */
+export const getMessage = (key) => currentLocale[key] || `{${key}}`;
```
Comment (milasevicius) [inline src/util/localization.js]: `{${key}}` brackets are not needed here

File: .gitignore (hunk 1 above)
Comment (Edvinas01) [reply .gitignore]: Its useful when checking out the project on different machines, you wont have to carry over and share global `.gitignore` file.

File: src/components/app/App.js (hunk 3 above)
Comment (Edvinas01) [reply src/components/app/App.js]: Debatable, this will make it hard to search for these files.

File: src/util/localization.js (hunk 7 above)
Comment (Edvinas01) [reply src/util/localization.js]: Brackets are needed here as desired result string is `'{some_invalid_key_here}'`.

**Pull request merged**